### PR TITLE
adding finetuning llms notebook

### PR DESCRIPTION
**Pull request by pavanbelagatti**

### CODE REVIEW

**Review by kesmit13 — CHANGES_REQUESTED** [review]
You need to run the pre-commit checks on your fork. The notebook JSON code doesn't look formatted properly.

I'm not sure how this notebook fits in with the rest of our collection. I don't see that SingleStoreDB is even used. Is there some connection that needs to be made to our software here?

Other issues:

1. You have a typo in the first line of the first section: "Singn" should be "Sign on"
2. You should put a link to the Gradient web site where the sign up is required.
3. The end of the first paragraph has some wording issues as well. The words in parens I believe should be "(we use them later on in this tutorial)". There is also a missing period.
4. You should add spaces around the '=' in the cell setting the env vars.
5. In the paragraph about fine-tuning the base model, you use the word "hallucinated". I'm not quite sure what you mean by that. Is there another word you could use? Maybe "unintelligible"?
6. There should commas after 'complete' and 'relevant' in the second to the last sentence in that same paragraph.
7. You should remove the `main` function from the last cell and split the cell into more digestible chunks, then add some descriptions to each step of the process.

**Comment by pavanbelagatti** [comment]
@kesmit13 Can I please get some help here to push this one?

**Comment by kesmit13** [comment]
@pavanbelagatti I posted some comments about fixes that need to be addressed yet.
